gs1/gs1.py

```diff
@@ -0,0 +1,102 @@
+"""Helper functions for GS1 barcode standard."""
+from itertools import cycle
+
+CHECK_DIGIT_FACTORS = (3, 1)
+# Defaults.
+APP_ID = "(00)"
+EXT = "0"
+# TODO: replace this with dataclass, once we can drop 3.6 support.
+BARCODE_LENGTHS = {
+    "gtin-8": 7,
+    "gtin-12": 11,
+    "gtin-13": 12,
+    "gtin-14": 13,
+    "gsin": 16,
+    "sscc": 17,
+
+}
```

Comment:
Unnecessary empty line?
```suggestion
    "sscc": 17,
}
```